agent-bootstrapper/build.gradle

```diff
@@ -18,6 +18,9 @@ import com.thoughtworks.go.build.VerifyJarTask
 
 description = 'GoCD Agent Bootstrapper'
 
+sourceCompatibility = 8
```

Comment:
I'd suggested that we do this for `jar-class-loader` module, and not this module.